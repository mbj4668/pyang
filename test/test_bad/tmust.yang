module tmust {

  namespace "http://netconfcentral.org/ns/tmust";

  prefix tm;

  revision "2015-10-08" {
  }

  leaf A { type int8; }

  container X {
    choice Y {
      must "/A=4";
      case YY {
        leaf ZZ { type string; }
      }
    }
  }
}
