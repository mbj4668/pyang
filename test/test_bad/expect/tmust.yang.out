tmust.yang:14: error: UNEXPECTED_KEYWORD
